lib/python/picongpu/plugins/data/saxsReader.py

```diff
@@ -0,0 +1,92 @@
+
+"""
+This file is part of the PIConGPU.
+
+Copyright 2017-2019 PIConGPU contributors
+Authors: Pawel Ordyna
+License: GPLv3+
+"""
+from .base_reader import DataReader
+
+from os import path
+import numpy as np
+import openpmd_api as api
```

Comment:
Please add a new line to `lib/python/picongpu/plugins/data/requirements.txt` which reads:
```
openPMD-api>=0.10.3
```